pt-BR/mail/chrome/messenger/focusPane.dtd

```diff
@@ -1,73 +1,73 @@
-<!ENTITY focusPaneSearchAttributeMac.label     "ATTRIBUTES">
-<!ENTITY focusPaneHasAttachmentAttribute.label           "Has Attachment">
-<!ENTITY focusPaneUnreadAttribute.label           "Unread">
-<!ENTITY focusPaneFilterByDate.label    "Date">
-<!ENTITY focusPaneFilterByDateMac.label "DATE">
-<!ENTITY focusPaneAnyDate.label         "Any Date">
-<!ENTITY focusPaneToday.label           "Today">
-<!ENTITY focusPaneYesterday.label       "Yesterday">
-<!ENTITY focusPaneThisWeek.label        "This Week">
-<!ENTITY focusPaneThisMonth.label       "This Month">
-<!ENTITY focusPaneFilterByContact.label    "Favorite Contacts">
-<!ENTITY focusPaneFilterByContactMac.label "FAVORITE CONTACTS">
+<!ENTITY focusPaneSearchScopeTitle.label       "Pesquisar">
+<!ENTITY focusPaneSearchScopeTitleMac.label    "PESQUISAR">
+<!ENTITY focusPaneSearchScopeEverywhere.label  "Todas as contas">
+<!ENTITY focusPaneSearchAttribute.label        "Atributos">
+<!ENTITY focusPaneSearchAttributeMac.label     "ATRIBUTOS">
+<!ENTITY focusPaneHasAttachmentAttribute.label           "Tem anexo">
+<!ENTITY focusPaneUnreadAttribute.label           "Não-lido">
+<!ENTITY focusPaneFilterByDate.label    "Data">
+<!ENTITY focusPaneFilterByDateMac.label "DATA">
+<!ENTITY focusPaneAnyDate.label         "Qualquer data">
+<!ENTITY focusPaneToday.label           "Hoje">
+<!ENTITY focusPaneYesterday.label       "Ontem">
+<!ENTITY focusPaneThisWeek.label        "Esta semana">
+<!ENTITY focusPaneThisMonth.label       "Este mês">
+<!ENTITY focusPaneFilterByContact.label    "Contatos favoritos">
+<!ENTITY focusPaneFilterByContactMac.label "CONTATOS FAVORITOS">
 
 <!-- focus pane topics context menus -->
-<!ENTITY pbFavoriteTopicsContextMenu-edit.label         "Rename">
+<!ENTITY pbFavoriteTopicsContextMenu-edit.label         "Renomear">
 <!ENTITY pbFavoriteTopicsContextMenu-edit.accesskey     "n">
-<!ENTITY pbFavoriteTopicsContextMenu-favorite.label     "Remove From Favorites">
+<!ENTITY pbFavoriteTopicsContextMenu-favorite.label     "Remover dos favoritos">
 <!ENTITY pbFavoriteTopicsContextMenu-favorite.accesskey "R">
-<!ENTITY pbFavoriteTopicsContextMenu-delete.label       "Delete">
+<!ENTITY pbFavoriteTopicsContextMenu-delete.label       "Excluir">
 <!ENTITY pbFavoriteTopicsContextMenu-delete.accesskey   "D">
-<!ENTITY pbFavoriteTopicsContextMenu-add.label          "Create New Favorite Topic">
+<!ENTITY pbFavoriteTopicsContextMenu-add.label          "Criar novo tópico favorito">
 <!ENTITY pbFavoriteTopicsContextMenu-add.accesskey      "C">
-<!ENTITY pbFavoriteTopicsContextMenu-editAll.label      "Edit Topics...">
+<!ENTITY pbFavoriteTopicsContextMenu-editAll.label      "Editar tópicos...">
 <!ENTITY pbFavoriteTopicsContextMenu-editAll.accesskey  "t">
 
 <!-- focus pane favorite contacts context menus -->
-<!ENTITY pbFavoriteContactsContextMenu-delete.label     "Remove From Favorites">
+<!ENTITY pbFavoriteContactsContextMenu-delete.label     "Remover dos favoritos">
```

Comment:
CORRECTIONS NEEDED it should be <!ENTITY pbFavoriteContactsContextMenu-delete.label     "Excluir dos favoritos">